### ISSUE TITLE
"Issue" and "problem" are used extensively on MDN as exact synonyms, which is not universally accepted.

### ISSUE THREAD

**Issue by Jaiket** [mdn/content#31304]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps

### What specific section or headline is this issue about?

Found a content problem with this page?

### What information was incorrect, unhelpful, or incomplete?

"Issue" and "problem" are used interchangeably on MDN, which lacks clarity and causes confusion.

### What did you expect to see?

I expect to see only the word "problem", with occasionally "problem or issue" when a page's content may be at risk of causing issues with unacceptable speech.

### Do you have any supporting links, references, or citations?

"When you use too many synonyms, you start to lose clarity in your work. "
https://uwaterloo.ca/writing-and-communication-centre/blog/power-synonyms-good-bad-and-between

https://www.quora.com/Diction-and-Word-Usage-Why-do-people-say-issue-when-they-really-mean-problem
https://learningenglish.voanews.com/a/problem-issue-or-matter-/5512446.html
https://langeek.co/en/grammar/course/1280/issue-vs-problem
http://www.differencebetween.net/language/difference-between-issue-and-problem/
https://www.quora.com/Whats-the-difference-between-the-word-issue-and-problem-in-English
https://thecontentauthority.com/blog/issue-vs-problem
https://www.grammarbook.com/blog/definitions/tackling-more-tricky-word-choices-issue-vs-problem/

### Do you have anything more you want to share?

I was unable to connect some content which was intended to directly refer to the title of a box :
Title : Found a content problem with this page?
Content : Report the content issue.

It took a good deal of explaining for a contributor to make me realise that the content was referring to the title.
I was convinced that there was a grammatical error. Since content "issues" had not been previously mentioned (although "content problems" had), I tried to correct the link to read "report a content issue".
As seen in the discussion below, I am not the only person to have found this confusing.
https://github.com/mdn/content/issues/30550#issuecomment-1826616867

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/css/first_steps`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/first_steps/index.md
* Last commit: https://github.com/mdn/content/commit/751d58669499de0c6ea0d5b356e0e1448418c5d3
* Document last modified: 2023-06-30T06:21:35.000Z

</details>

**Comment by hamishwillee** [comment]
Can someone with rights, please move this to yari repo?

The post is a little confusing, but I believe that they think the use of "issue" in "report the content issue" is inconsistent with the title:.
![image](https://github.com/mdn/content/assets/5368500/a2b35b49-c46d-48bd-abf6-e432d76c456d)

FWIW I think this is not at all confusing, but perhaps others will disagree.

**Comment by Jaiket** [comment]
@hamishwillee You understod me correctly.
My post is about whether issue and problem are exact synonyms or not. The links I provided claim they are not.
Can you point me towards what was confusing about my post?

**Comment by hamishwillee** [comment]
> Can you point me towards what was confusing about my post?

Yes, it took me several reads to work out that you were talking about the auto-generated links at the bottom of each page. Sometimes it is best to just show a screenshot.
